### PR TITLE
Tm 4621 anderson documentation storks

### PR DESCRIPTION
**Pull request by anderolimar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,163 @@
-# iTaaS Storks
+<p align="center">
+  <img src=".assets/storks.png" alt="Storks" width="150">
+  <br>
+</p>
 
-[![Build Status](https://travis-ci.org/UUX-Brasil/storks.svg?branch=master)]
-(https://travis-ci.com/UUX-Brasil/storks)
+<h4 align="center"> 
+Storks is a agent to delivery log files from rsyslog to AWS S3. <br>
```
Comment (rzamana) [inline README.md]: "Storks is **an** agent"

File: README.md [hunk 2]
```diff
@@ -1,55 +1,163 @@
...
-* Log files ended with '-YYYYMMDDhhmm.log';
+## 1. Requirements
 
-----
-## Development
+* <img src=".assets/linux.png" alt="Linux" align="center" width="30">  [Linux](https://www.linux.org/pages/download/) / <img src=".assets/windows.png" alt="Linux" align="center" width="30">  [Windows](https://www.microsoft.com/en-us/windows/) 
+* <img src=".assets/docker.png" alt="Docker" align="center" width="30">  [Docker](https://www.docker.com) 
+* <img src=".assets/node.png" alt="Node" align="center" width="30"> [Node.js v6.9+](https://nodejs.org) 
+* <img src=".assets/npm.png" alt="npm" align="center" width="30">  [npm v3.10+](https://www.npmjs.com/)  
+* <img src=".assets/vscode.png" alt="Visual Studio Code" align="center" width="30">  [Visual Studio Code](https://code.visualstudio.com/)
```
Comment (rzamana) [inline README.md]: I don't think Visual Studio Code is a requirement, it's possible to edit the code in others IDE

File: README.md [hunk 3]
```diff
@@ -1,55 +1,163 @@
...
+Start Storks with rsyslog (Linux) running follow command.
+
+*You must have rsyslog installed.*
+
+```
+npm run start-with-rsyslog
+```
+## 4. Documentation
+
+All documentation is in this file.
```
Comment (rzamana) [inline README.md]: Remove this section, it's not necessary!

File: README.md [hunk 4]
```diff
@@ -1,55 +1,163 @@
...
+
 ```
-for a straight run,
-or use
+code .
 ```
+
+### 5.1. Building
+
+You don't need build node.js scripts.
```
Comment (rzamana) [inline README.md]: Remove this section, it's not necessary.

File: README.md [hunk 5]
```diff
@@ -1,55 +1,163 @@
...
+
+### 5.2. Debugging
+
+Each debug mode will listen to port 5859. Just attach your favorite debugger tool to port 5859, after running a debug command.
```
Comment (rzamana) [inline README.md]: Here we use **favorite** debugger, but above we must use CODE.

File: README.md [hunk 6]
```diff
@@ -1,55 +1,163 @@
...
+Run all integration tests with command bellow:
+
+```
+npm run integration-test
+```
+
+Run eslint tests with command bellow:
+
+```
+npm run lint
```
Comment (rzamana) [inline README.md]: It's possible to run **EVERY** tests in a single command?

File: README.md [hunk 7]
```diff
@@ -1,55 +1,163 @@
...
+```
+
+#### 5.3.1 Travis
+
+Travis CI is a hosted, distributed continuous integration service used to build and test software projects hosted at GitHub.
```
Comment (rzamana) [inline README.md]: Put the link of the travis homepage.

Is Travis a requirement?

File: README.md [hunk 8]
```diff
@@ -1,55 +1,163 @@
...
+You can build a Docker image while in the root folder like this:
+
+```
+docker build -t storks .
+```
+
+Then, in order to start Storks, you can simply run
+
+```
+docker run -d storks
```
Comment (rzamana) [inline README.md]: Remove the daemon option, if it's an unskilled person trying to run, he will have storks running forever.

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+  1. [New Feature](#new-feature)
+  2. [Branching](#branching)
+  3. [Commiting](#commiting)
+  4. [Pull Requests](#pull-requests)
+  5. [Additional Resources](#additional-resources)
+
+## New Feature
+  
+  Every new feature must have a test associated with it. Either an unit test or an integration test.
```
Comment (rzamana) [inline CONTRIBUTING.md]: "An unit test **AND** an integration test"

File: README.md [hunk 10]
```diff
@@ -1,55 +1,163 @@
...
+*You must have rsyslog installed.*
+
+```
+npm run start-with-rsyslog
+```
+## 4. Documentation
+
+All documentation is in this file.
+
+## 5. Development
```
Comment (rzamana) [inline README.md]: We use semver, talk about it!

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+  ```<TM-XXXX>_<name>_<brief-description>```
+
+## Commiting
+
+  Your commit messages must describe briefly what was done. Try to make your commit messages in one line.
+
+## Pull Requests
+
+  1. Your branch must pass at Travis-CI.
+  2. Your branch must have updated the CHANGELOG.md. It must follow the standard below:
```
Comment (rzamana) [inline CONTRIBUTING.md]: 3. Include in the CHANGELOG.md if your branch adds:
- a bugfix (*patch*)
- or a new feature (*minor*)

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+  
+  ```<TM-XXXX>_<name>_<brief-description>```
+
+## Commiting
+
+  Your commit messages must describe briefly what was done. Try to make your commit messages in one line.
+
+## Pull Requests
+
+  1. Your branch must pass at Travis-CI.
```
Comment (rzamana) [inline CONTRIBUTING.md]: 2. Your *PR* must be set to base **dev**

File: README.md (hunk 1 above)
Comment (anderolimar) [inline README.md]: ok

File: README.md (hunk 3 above)
Comment (anderolimar) [inline README.md]: ok

File: README.md (hunk 2 above)
Comment (anderolimar) [inline README.md]: ok

File: README.md (hunk 4 above)
Comment (anderolimar) [inline README.md]: ok

File: README.md (hunk 5 above)
Comment (anderolimar) [inline README.md]: ok

File: README.md (hunk 6 above)
Comment (anderolimar) [inline README.md]: No

File: README.md (hunk 7 above)
Comment (anderolimar) [inline README.md]: Travis is not a requirement

File: CONTRIBUTING.md (hunk 9 above)
Comment (anderolimar) [inline CONTRIBUTING.md]: Sometimes you can have only one test type.

File: README.md (hunk 8 above)
Comment (anderolimar) [inline README.md]: ok

File: README.md (hunk 10 above)
Comment (anderolimar) [inline README.md]: Done .

File: README.md (hunk 7 above)
Comment (anderolimar) [inline README.md]: Link added.

File: CONTRIBUTING.md (hunk 12 above)
Comment (anderolimar) [inline CONTRIBUTING.md]: It's not a rule, you can have PR's from branches that is not dev.

File: CONTRIBUTING.md (hunk 11 above)
Comment (anderolimar) [inline CONTRIBUTING.md]: Done differently.

File: README.md [hunk 13]
```diff
@@ -1,55 +1,159 @@
...
 
-----
-## Features
+* [1. Requirements](#1-requirements)
+* [2. Installation](#2-installation)
+* [3. Usage](#3-usage)
+* [4. Development](#4-development)
+  * [4.1. Debugging](#41-debugging)
+  * [4.2. Testing](#42-testing)
+  * [4.2. Version](#43-version)
```
Comment (rzamana) [inline README.md]: There is two 4.2

File: README.md [hunk 14]
```diff
@@ -1,55 +1,159 @@
...
+In folder scripts, has two scripts for travis: 
+
+* travis-step-install.sh
+  * Install storks using `npm install` command.
+* travis-step-script.sh 
+  * Run tests and create docker image.
+
+This scripts is defined in [.travis.yml](.travis.yml)
+
+You can see our Travis environment [here](https://travis-ci.com/agilecontent).
```
Comment (rzamana) [inline README.md]: For this project our Travis is in .org URL

File: README.md (hunk 13 above)
Comment (anderolimar) [inline README.md]: ok

File: README.md (hunk 14 above)
Comment (anderolimar) [inline README.md]: ok

**Pull request merged**